fe1-web/src/core/components/RoundIconButton.tsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, TouchableOpacity, ViewStyle } from 'react-native';
+import { Icon } from 'react-native-elements';
+
+const styles = StyleSheet.create({
+  roundButton: {
+    height: 65,
+    width: 65,
+    shadowColor: 'gray',
+    shadowOffset: {
+      width: 2,
+      height: 2,
+    },
+    borderRadius: 30,
+    backgroundColor: '#2196F3',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  } as ViewStyle,
+});
+
+/**
+ * This is a round button with an icon that uses Google material icons
+ * The 'name' property is the name of the icon that will be rendered,
+ * you can look for icon and their names here : https://fonts.google.com/icons
+ */
+const RoundIconButton = (props: IPropTypes) => {
+  const { name, onClick } = props;
+  return (
+    <TouchableOpacity style={styles.roundButton} onPress={onClick}>
+      <Icon name={name} type="material" color="white" size={25} />
+    </TouchableOpacity>
+  );
+};
```

Comment:
This is design choice but I would argue it might make sense to define several Icon components. Your version is much more general but this "allows" the usage of many many different icons. So maybe not export this component directly but rather export `SettingsIcon`, `BackIcon`, etc. that fix the icon used (i.e. the "name"). This will make it harder to use arbitrary icons / easier to use the existing ones. We should try to keep the used icons consistent.

@pierluca @Xelowak  What do you think?

Reply:
I agree with you. Having icons that are directly mapped to their functionality is easier to understand, and it remains consistent easier throughout the application.